src/components/Grid/ActivationDataFormatter.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ProgressBarIndicator from '../progressBar/progressBarIndicator';
+import Icon from '../Icon';
+
+
+const ActivationDataFormatter = ({ row }) => {
+    if (row.activationData === 100) {
```

Comment:
Can we somehow extract the `50/100` magic numbers?

Reply:
Fixed